tox/constants.py

```diff
@@ -0,0 +1,56 @@
+"""All non private names (no leading underscore) here are part of the tox API.
+
+They live in the tox namespace and can be accessed as tox.[NAMESPACE.]NAME
+"""
+import sys as _sys
+
+
+class PYTHON:
+    CPYTHON_VERSION_TUPLES = [(2, 7), (3, 4), (3, 5), (3, 6), (3, 7)]
+    OTHER_PYTHON_INTERPRETERS = ['jython', 'pypy', 'pypy3']
+    _map = {'py': _sys.executable, 'py2': 'python2', 'py3': 'python3'}
```

Comment:
would prefer to have a single constructor function that's private and then call that once, as ``X = _construct)_``, then this line by line alteration, makes me not sure if these consts are consts this way

Reply:
agreed, made a little helper function.